src/model/Role.ts

```diff
@@ -0,0 +1,35 @@
+import BaseEntity, { BaseEntityArgs } from './BaseEntity';
+
+export interface ProviderRoleDetails {}
+
+export interface KeycloakRoleRepresentation extends ProviderRoleDetails {
+  id?: string;
+  name?: string;
+  description?: string;
+  scopeParamRequired?: boolean;
+  composite?: boolean;
+  // TODO Recheck this type
```

Comment:
Did you plan to resolve this TODO before merging?

Reply:
Good catch! Fixed in fff616582c93f92790642c9eada50c04d1264a17.